src/api/app/views/webui/shared/_download_repository_link.html.haml

```diff
@@ -1,5 +1,10 @@
 - url = "#{download_url}/#{project.to_s.gsub(/:/, ':/')}/#{repository}"
-%li.list-inline-item
-  = link_to(url, title: 'Go to download repository', class: 'nav-link') do
-    %i.fas.fa-download
-    Go to download repository
+- download_area_exist = Backend::Api::Published.download_area_exist?(project, repository)
```

Comment:
view doing a query direct to the backend?

Reply:
Moved into a presenter